jest.config.ts

```diff
@@ -32,7 +32,7 @@ const config: Config = {
     // ms to wait before throwing a timeout error
     testTimeout: 60_000,
     json: true,
-    maxWorkers: 3,
+    maxWorkers: 10,
```

Comment:
Can we revert this back to 3? Tests may no be clean since we're sending lots of transactions.

Reply:
From the tests I did, running with 10 workers behave the same way as running with 3 workers, lowered to 3 anyway in the commit 4d987ae4c481425a7ba9a32016c93c9569ae8ab4